### DISCUSSION TITLE
"Hobby teams can enable Speed Insights on a single project at a time"

### DISCUSSION THREAD

**Discussion by Mohamed3on** [vercel/community#7224] [Help] [answered]
### Summary

Hi,
I've got a few hobby projects on Vercel. Whenever I try to enable Speed insights on a project, I get the above error. 

When I go to the project where Speed Insights are supposed to be enabled, I still see the same message:
<img width="1206" alt="image" src="https://github.com/vercel/community/assets/12295159/e99c7b2a-e1bf-44c2-b152-31542deb9906">

Are Speed Insights still available on Hobby plans?

Thanks!

### Steps to Reproduce

Try to disable speed insights on a project that had it enabled, and you'll see this: 
<img width="1206" alt="image" src="https://github.com/vercel/community/assets/12295159/16312b6c-c31b-4390-8edf-71bfdaad171d">

**Comment by leerob** [comment, marked as answer]
Hey there. Hobby teams (free) can only enable Speed Insights on a single project. If you would like to use multiple projects, you can upgrade to Pro!

**Reply by Mohamed3on** [reply]
@leerob I understand this, my issue is that I can't enable it even though I don't have it enabled in any project, does that make sense?

**Reply by amyegan** [reply]
Hey @Mohamed3on. The error message means that you already have it enabled on one of your projects. Please check your other projects to find the one where it is currently enabled. If you disable it there, then you will be able to add it to a different project.

**Reply by anoop0567** [reply]
@amyegan Is there any way to know which projects have enabled Speed Insights in my hobby projects? I have multiple projects, and going through each project and checking will take a long time.

**Reply by amyegan** [reply]
The simplest way to identify which projects have it enabled is the speed insights indicator icon. You can see a version of it on each project in the list.

Projects with speed insights enabled will look something like this.
<img width="162" alt="colorful circle gauge with number" src="https://github.com/user-attachments/assets/89d4835d-c4b7-498b-97bb-c7d88d9ea009">

Projects _without_ speed insights enabled will look like this.
<img width="136" alt="gray circle gauge with pulse icon" src="https://github.com/user-attachments/assets/d69f58d8-e7c6-43fb-844f-d2cc26220500">

[a comment by an automated account is not shown]
